package.json

```diff
@@ -13,5 +13,10 @@
   },
   "ember-addon": {
     "main": "ember-addon-index.js"
+  },
+  "devDependencies": {
```

Comment:
these are all used at runtime, and need to be dependencies (not dev deps)

Reply:
Haha, your comment was not destroyed in the renaming.  I have reset the dependencies as you suggested on master.  :beers: